api/test/integration/test_manager_endpoints.tavern.yaml

```diff
@@ -973,47 +973,6 @@ stages:
           total_affected_items: 1
           total_failed_items: 0
 
----
-test_name: GET /manager/validation (KO)
```

Comment:
This test was failing because the corrupted file couldn't be uploaded, the `PUT /rules/files` endpoint rejected it.

Reply:
I wonder how was this working before? I tried to upload an invalid configuration also in 4.7.3 and, even if the status code is 200, the task is shown as a failed item and then, the validation should be successful:

![image](https://github.com/wazuh/wazuh/assets/23361101/72035020-5ab7-45bf-b79c-5cd678d5417e)

Could you review this just so we understand it better?

Reply:
This is because the corrupted rule file was considered valid before the changes in this [PR](https://github.com/wazuh/wazuh/pull/20507).

It was added back again [here](https://github.com/wazuh/wazuh/commit/4d1cafeaa54f7b75fdf5fa01c5a71bcc9e3d4f68#diff-1762fd009e02995fa8cac49665efd33fcefae496f406b45902be89c0b75b66d1) by mistake.